publish.ps1

```diff
@@ -68,10 +68,12 @@ function do-package()
     Copy-Item -Force "$distDir\readme.txt"            $tempDir -ErrorAction Stop
 
     # Create archive
-    7za a -tzip -mx9 $outZip "$tempDir\*"
+    7z a -tzip -mx9 $outZip "$tempDir\*"
```

Comment:
That's what their docs say: http://www.appveyor.com/docs/packaging-artifacts#packaging-multiple-files-in-different-locations-into-a-single-archive